### PR TITLE
adds group inactive tag

### PR DESCRIPTION
**Pull request by sareg0**
@Malwine and I worked on the issue #298 about having an 'group inactive' tag, so that a person knows if the group inactive before joining.

The tests are running, but another pair of eyes, to see if it's missing anything is always good.

### CODE REVIEW

**Comment by berlintam** [comment]
:+1: looks good! I guess you'll have to merge master into it thou.

**Comment by sareg0** [comment]
oh gosh... okay, I'll fix that up then push the changes. Probs end of day.

**Comment by berlintam** [comment]
you can do it! (lemmie know if u need help)

**Comment by sareg0** [comment]
@berlintam did I do it correctly? Had to use Tower and Marc, but got it :)

**Comment by berlintam** [comment]
looks pretty good to me! (let's see what Travis says...)

**Comment by sareg0** [comment]
Travis says go!

File: Gemfile.lock [hunk 1]
```diff
@@ -340,6 +340,3 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   will_paginate
-
-BUNDLED WITH
-   1.10.5
```
Comment (berlintam) [inline Gemfile.lock]: my only concern is that this needs to be back in the gemfile.lock.

Brigitte had the same problem on one of her branches (https://github.com/rubycorns/rorganize.it/pull/350), and @til  said:

"Just revert this change to this file, and push that as new commit to this branch?

Bonus points for squashing it, but can be done later.

Oh, and run a gem install bundler locally to make sure you have the most recent one installed. Then you should be safe from getting this change applied automatically again, I think there were a few recent versions of bundler that always changed this section, but the most recent one only does it if the Gemfile.lock also changed elsewhere."

Comment (berlintam) [inline Gemfile.lock]: I think this stackoverflow comment can help you with that...
http://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git

**Comment by PragTob** [comment]
:+1:

**Comment by Malwine** [comment]
giphy

**Pull request merged**